graphql-dgs-spring-webflux-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/webflux/handlers/DgsReactiveWebsocketHandler.kt

```diff
@@ -74,9 +74,10 @@ class DgsReactiveWebsocketHandler(private val dgsReactiveQueryExecutor: DgsReact
                             )
                         )
                         GQL_START -> {
-                            val queryPayload =
-                                jacksonObjectMapper().convertValue(operationMessage.payload, QueryPayload::class.java)
-                            logger.debug("Starting subscription $queryPayload for session ${webSocketSession.id}")
+                            val queryPayload = decoder.objectMapper.convertValue<QueryPayload>(
+                                operationMessage.payload ?: error("payload == null")
```

Comment:
any debugging context worth logging about the caller?

Reply:
I am not sure what the correct thing to do here is; this is really just making explicit the behavior of the existing code, which would have passed the null value silently and blown up a few lines down with an NPE when it tried to access `queryPayload.query`. I guess logging a warning could make sense here.